Shared/DataTransferObjects/AttendanceDto.cs

```diff
@@ -0,0 +1,9 @@
+namespace Shared.DataTransferObjects;
+
+public record AttendanceDto(
+    Guid Id, 
+    Guid EmployeeId, 
+    DateTime CheckInTime, 
+    DateTime? CheckOutTime, 
+    string Status, 
```

Comment:
Consider annotating the `Status` property as nullable (e.g., `string?`) if it is not mandatory. If it is required, ensure that validation enforces it in a separate validator class. As per nullable reference type guidelines, explicit types clarify intent.

```suggestion
    string? Status,
```